src/encoded/static/components/publication.js

```diff
@@ -38,16 +38,11 @@ const datasetsColumns = {
         title: 'Description',
     },
     lab: {
-        getValue: dataset => (dataset.lab || {}).title,
+        getValue: dataset => (dataset.lab ? dataset.lab.title : ''),
         title: 'Lab',
     },
     biosample_summary: {
-        getValue: (dataset) => {
-            if (typeof (dataset.biosample_summary) === 'string') {
-                return dataset.biosample_summary;
-            }
-            return (dataset.biosample_summary || []).map(b => b.term_name).join(', ');
-        },
+        getValue: dataset => dataset.biosample_summary,
```

Comment:
Since you’re getting the property with the same name as they key (`biosample_summary`) you don’t need `getValue`. Just `title` is enough because SortTable will figure it out.

Reply:
Done